docker-compose-extra.yml

```
@@ -174,6 +176,19 @@ services:
+    hostname: swmath-swh
+    image: *swmath2swh-image
+    environment:
+    # start over on each run
+    - OAI_STATE_PATH=/tmp/state.json
+    - SWH_API_TOKEN
+    - SWMATH_USER_DEPOSIT_PRODUCTION
+    - SWMATH_PWD_DEPOSIT_PRODUCTION
+    depends_on:
+    - oai-backend
+    restart: always
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)